en/docs/reference/connectors/file-connector/file-connector-config.md

```diff
@@ -1128,6 +1128,35 @@ The following operations allow you to work with the File Connector version 4. Cl
                 Yes
             </td>
         </tr>
+        <tr>
+            <td>
+                Disk Share Access Mask
+            </td>
+            <td>
+                diskShareAccessMask
+            </td>
+            <td>
+                String
+            </td>
+            <td>
+                <p> The `diskShareAccessMask` configuration applies when using the **SMB2** or **SMB3** protocol for file operations through the file connector. This setting defines the access permissions (ACCESS_MASK) for the disk share. Choose a value that matches your security policies and operational needs. </p>
+                Supported `diskShareAccessMask` Values:
+                <ul>
+                <li>**READ**: Grants read-only access.</li>
+                <li>**WRITE**: Grants write-only access.</li>
+                <li>**READ_WRITE**: Grants both read and write access.</li>
+                <li>**ALL**: Grants full control over the file share.</li>
+                <li>**MAXIMUM**: Grants the maximum permissions allowed.</li>
+                </ul>
+                <p>**Note:** Requires server version 4.4.0.3 (U2) or later to use this parameter.</p>
```

Comment:
Shall we add a note block here.

Reply:
Since we mention about file servers in this doc, better to mention this is the MI server version.